In [1]:
import pandas as pd 

In [2]:
from sklearn.linear_model import LinearRegression

In [3]:
df = pd.read_csv("C:/Users/manuell/Videos/CienciaDatos_AN/datosmexico.csv")

In [4]:
df.head()

Label  PIBperCapita  FuerzaLaboral
0   2019  10145.152630      64.426002
1   2018   9857.007487      64.346001
2   2017   9434.378179      64.043999
3   2016   8875.082069      64.164001
4   2015   9753.400562      64.135002

In [5]:
# Definicion de Variable Independiente y Dependiente

In [6]:
x = df[["FuerzaLaboral"]]

In [7]:
y = df[["PIBperCapita"]]

In [9]:
# Creacion de la instancia del modelo

In [10]:
modelo = LinearRegression()

In [11]:
# Ajuste del modelo

In [12]:
modelo.fit(x,y)

LinearRegression()

In [13]:
interceptor = modelo.intercept_

In [14]:
coeficiente = modelo.coef_[0]

In [16]:
interceptor

array([-88928.00961016])

In [17]:
coeficiente

array([1532.42107987])

#### Explicacion

##### Ecuacion de la **Regresion Lineal**
**PIB_per_Capita** = 1532.42 * *Fuerza Laboral* - 88928

El coeficiente de la variable *Fuerza laboral* es igual a **$1,532.42**, lo que implica que por cada punto porcentual que aumenta la taza de participacion  en la fuerza laboral, se espera que el PIB aumente  aprox *1,532.42*